src/Navbar.tsx

```diff
@@ -1,13 +1,99 @@
+import { NavLink } from "react-router-dom";
+import logo from "../public/img/logo.png";
+import { useState, useEffect, useRef, LegacyRef } from "react";
+import { LuMenu } from "react-icons/lu";
+import { CgClose } from "react-icons/cg";
+
 const Navbar = () => {
-    return (
-    <div className = "fixed"> 
-        <ol>
-            <li><a href="/">home</a></li>
-            <li><a href="/menu">menu</a></li>
-            <li><a href="/about">about</a></li>
-        </ol>
-    </div>
-    )
-  }
-
-export default Navbar
\ No newline at end of file
+  const [isSideBarOpen, setIsSideBarOpen] = useState<boolean>(false);
+  let sidebarRef: LegacyRef<HTMLDivElement> = useRef(null);
+
+  useEffect(() => {
+    const handleClickOutside = (e: MouseEvent) => {
+      if (
+        sidebarRef.current &&
+        !sidebarRef.current.contains(e.target as Node)
+      ) {
+        setIsSideBarOpen(false);
+      }
+    };
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, [sidebarRef]);
+
+  return (
+    <nav className="fixed w-full z-20 mt-[33px]">
+      <div className="flex flex-wrap justify-between items-center mx-[6%]">
+        {/* LOGO */}
+        <NavLink to="/">
+          <img src={logo} alt="poke-bay-logo" className="w-[55px] h-[55px]" />
+        </NavLink>
+
+        {/* MOBILE MENU */}
```

Comment:
Please left-align the links and right-align the close button as shown here
<img width="302" alt="image" src="https://github.com/webimpactuw/poke-bay/assets/25410985/9c473c1b-6575-46bc-b86c-a14aba1b47c6">